css/main.css

```diff
@@ -1,43 +1,85 @@
-body {
-	margin: 0;
-	padding: 0;
-	background-color: #000;
-}
-
-.master-logo {
-	position: absolute;
-	top: 0;
-	left: 0;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	width: 100%;
-	height:100%;
-}
-
-@keyframes spin {
-  from{
-    transform: rotateY(0deg);
-    moz-transform: rotateY(0deg); //Firefox
-    ms-transform: rotateY(0deg); //Microsoft Browsers
+*{
+  margin: 0;
+  padding: 0;
+}
+
+/********************************************\
+******************  HEADER  *****************
+/********************************************\*/
+
+.header-container{
```

Comment:
vsade pred zlozenou zatvorkou { musi byt medzera. je to coding standard. Prosim oprav to tak